feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/send/amount/SelectSendViewModel.kt

```diff
@@ -493,6 +504,33 @@ class SelectSendViewModel(
         }
     }
 
+    private suspend fun getMetaAccountsFilterPayload(origin: Chain, desination: Chain): SelectAddressRequester.Request.Filter {
+        val isCrossChain = origin.id != desination.id
+        return if (isCrossChain) {
+            SelectAddressRequester.Request.Filter.Empty
```

Comment:
I suggest to name such a filter `Everything` since `Empty` has unclear semantics imho